### DISCUSSION TITLE
delay for database notification

### DISCUSSION THREAD

**Discussion by jromainkrupa** [excid3/noticed#303] [Q&A] [answered]
Great Gem such a pleasure ! 
Junior question but is there a way to delay the creation of the notification for database delivery ? 
I saw that delay is not an option for database delivery and raise an error
I am about to create a `DelayNotificationCreationJob` to use `wait` but there is probably a good reason if it's not implemented ? 

The use case is I have a `ContentAssigned` notification to assign a content. users have a tendency to assign 3/4 contents and I want to wait 15.minutes to be able to batch into 1 notification.

**Comment by excid3** [comment, marked as answer]
Yep, that's what you should do. The other notification methods depend on the record, so it must be created first.

Maybe we'll add a built-in job for that.

**Comment by jromainkrupa** [comment]
Thank you so much for your answer and the gem and all the amazing stuff you’re doing
